frontend/app/(home)/components/Sections/DemoSection/components/VideoPlayer.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useRef, useState } from "react";
+
+interface VideoPlayerProps {
+  videoSrc: string;
+}
+
+export const VideoPlayer = ({ videoSrc }: VideoPlayerProps): JSX.Element => {
+  const videoRef = useRef<HTMLVideoElement>(null);
+  const [isPlaying, setIsPlaying] = useState(false);
```

Comment:
WDYT https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/paused ?

Reply:
You're right! I just committed some changes accordingly!